source/_docs/local-development.md

```diff
@@ -159,26 +160,32 @@ Send files using SFTP:
 You can also transfer a single file or a single directory at a time instead of transferring every file, every time.
 
 ## Local Configuration Files
+
 You'll need to configure database credentials matching your local database to develop locally. You don't want to manually change these details in your primary configuration file (e.g. `settings.php` or `wp-config.php`) because you could easily commit that change to version control and trigger a connection error on Dev when pushing to Pantheon.
 
 Instead, we recommend using a local configuration file (e.g. `settings.local.php` or `wp-config-local.php`) that is excluded from version control and included by `settings.php` or `wp-config.php` when found. Since the local configuration file is ignored by git, it won't be found on Pantheon but it will be applied when you run the site locally.
 
-Pantheon's Drupal 8 and WordPress upstreams apply `settings.local.php` or `wp-config-local.php` when found and ignore them from git without additional steps required. All you need to do is create a `settings.local.php` or `wp-config-local.php` file then use it to configure your local setup.
+### Drupal 8 and WordPress
 
-However, Drupal 7 users will need to include it within their `settings.php` file:
+Pantheon's [Drupal 8](https://github.com/pantheon-systems/drops-8/blob/f1e72395cbc3bc76768eaec725a4e7fa18848cf3/sites/default/settings.php#L17) and [WordPress](https://github.com/pantheon-systems/WordPress/blob/344fc6e61998f44806d25eb6232dd068055b532b/wp-config.php#L18) upstreams apply `settings.local.php` or `wp-config-local.php` when found and ignore them from git without additional steps required. All you need to do is create a `settings.local.php` or `wp-config-local.php` file then use it to configure your local setup.
```

Comment:
Can we add links to the [Drupal 8 .gitignore](https://github.com/pantheon-systems/drops-8/blob/master/.gitignore#L8) and [WordPress .gitignore](https://github.com/pantheon-systems/WordPress/blob/master/.gitignore#L3) files?